examples/TF_1_x/object_detection_sample.py

```diff
@@ -239,6 +239,13 @@ def load_labels(label_file):
             img_resized = preprocess_image(image,
                                            (input_height, input_width))[0]
 
+            if (image_id == 0):
```

Comment:
As discussed, can we remove the changes to the samples, and add them as tutorials in the documentation? We can only keep the API functions **prepare_model** and **prepare_model_with_session** as it'd be confusing to include it in samples now, and it doesn't align with our "two lines of code" mantra.